crowbar_framework/lib/crowbar/error/upgrade_status.rb

```diff
@@ -29,8 +29,9 @@ def initialize(step_name = "")
     end
 
     class StartStepOrderError < StandardError
-      def initialize(step_name = "")
-        super("Start of step #{step_name} requested in the wrong order")
+      def initialize(step_name = "", correct_step = "")
+        super("Start of step '#{step_name}' requested in the wrong order. " \
+        "Correct next step is '#{correct_step}'.")
```

Comment:
Well, the concern is the same when `step_name` is empty, right? 
But I'm always passing some values to `StartStepOrderError`, so it it a problem? Maybe we should just drop the default values then?